### PR TITLE
Add py-fastcov

### PR DESCRIPTION
**Pull request by haampie**

### CODE REVIEW

File: var/spack/repos/builtin/packages/py-fastcov/package.py [hunk 1]
```diff
@@ -0,0 +1,24 @@
...
+    """
+
+    homepage = "https://github.com/RPGillespie6/fastcov"
+    pypi     = "fastcov/fastcov-1.13.tar.gz"
+
+    maintainers = ['haampie']
+
+    version('1.13', sha256='ec8a5271f90a2f8b894cb999e262c33e225ed6072d9a6ca38f636f88cc0543e8')
+
+    # Depends on gcov too, but that's installed with gcc
```
Comment (adamjstewart) [inline var/spack/repos/builtin/packages/py-fastcov/package.py]: Does this mean it only supports GCC?

Comment (haampie) [reply var/spack/repos/builtin/packages/py-fastcov/package.py]: Hm, let me rephrase that to `with the compiler`, cause I think llvm also has a gcov, but a different codecov format.

Comment (haampie) [reply var/spack/repos/builtin/packages/py-fastcov/package.py]:
```suggestion
    # Depends on gcov too, but that's installed with the compiler
```

**Review by adamjstewart — APPROVED** [review]
